mongodb_exporter.go

```diff
@@ -42,10 +42,11 @@ var (
 	listenAddressF = kingpin.Flag("web.listen-address", "Address to listen on for web interface and telemetry.").Default(":9216").String()
 	metricsPathF   = kingpin.Flag("web.telemetry-path", "Path under which to expose metrics.").Default("/metrics").String()
 
-	collectDatabaseF   = kingpin.Flag("collect.database", "Enable collection of Database metrics").Bool()
-	collectCollectionF = kingpin.Flag("collect.collection", "Enable collection of Collection metrics").Bool()
-	collectTopF        = kingpin.Flag("collect.topmetrics", "Enable collection of table top metrics").Bool()
-	collectIndexUsageF = kingpin.Flag("collect.indexusage", "Enable collection of per index usage stats").Bool()
+	collectDatabaseF            = kingpin.Flag("collect.database", "Enable collection of Database metrics").Bool()
+	collectCollectionF          = kingpin.Flag("collect.collection", "Enable collection of Collection metrics").Bool()
+	collectTopF                 = kingpin.Flag("collect.topmetrics", "Enable collection of table top metrics").Bool()
+	collectIndexUsageF          = kingpin.Flag("collect.indexusage", "Enable collection of per index usage stats").Bool()
+	mongodbCollectConnPoolStats = kingpin.Flag("collect.connpoolstats", "Collect MongoDB connpoolstats").Bool()
```

Comment:
Add `F` suffix as in other flags. Also make help text consistent with other `collect.XXX` flags.